### ISSUE TITLE
Add subscribe to the mailing list on the landing page

### ISSUE THREAD

**Issue by bethcg** [SwissDataScienceCenter/renku-ui#3288]
## Motivation

Renku has now its newsletter service ready: [Resend](https://resend.com/settings). We would like to have our users an easy way to subscribe to our newsletter. We would like to have a place in the landing place for a form that allows users to automatically subscribe (Javascript available).

**Comment by lorenzo-cavazzi** [comment]
I guess that will fit well on the landing page for anonymous users, but I'm not 100% sure we want to take up space for that in the Dashboard. I'm wondering if we want to show something in the user dropdown menu and have a section on the Help page.

P.S. should this be for both Renku 1.0 and 2.0?

More info for Resed: https://resend.com/docs/api-reference/contacts/create-contact

**Comment by bethcg** [comment]
> I guess that will fit well on the landing page for anonymous users
Agree, it should be on landing page or a new "subscribe" page accessible from a click in the landing page.

>, but I'm not 100% sure we want to take up space for that in the Dashboard. I'm wondering if we want to show something in the user dropdown menu and have a section on the Help page.
Agree that it does not make sense in the Dashboard. I could well foresee that we create an informing message when logging in for the first time with the possibility to register to the Newsletter, and then include it under Help, the same way as Gitter, Forum or GitHub.

> P.S. should this be for both Renku 1.0 and 2.0?
This should be both Renku 1.0 and 2.0

**Comment by lorenzo-cavazzi** [comment]
I'm closing this issue since it was collecting proposals but it's not actionable.
I'm happy to discuss the topic further and open a new one once we agree on a solution.
